### PR TITLE
Local On-boarding flow

### PR DESCRIPTION
**Pull request by acunniffe**
This PR contains the new onboarding flow -- it starts immediately after init on` /apis/{number}/setup`.

### CODE REVIEW

File: workspaces/cli-client/src/spec-service-client.ts [hunk 1]
```diff
@@ -51,12 +52,19 @@ export class Client implements ISpecService {
     );
   }
 
-  loadConfig(): Promise<{ config: IApiCliConfig }> {
-    return JsonHttpClient.getJson(
+  async loadConfig(): Promise<{ config: IApiCliConfig; configRaw: string }> {
+    return await JsonHttpClient.getJson(
```
Comment (devdoshi) [inline workspaces/cli-client/src/spec-service-client.ts:56]: awaiting should be up to the caller, this should just pass the promise through. Same for all of these, to be consistent.

File: workspaces/ui/scripts/fetch-docs.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const fetch = require('node-fetch');
```
Comment (devdoshi) [inline workspaces/ui/scripts/fetch-docs.js:1]: @notnmeyer thoughts?
@acunniffe I think this should be done more statically

Comment (acunniffe) [reply workspaces/ui/scripts/fetch-docs.js:1]: this is currently what we do for the saas site, but open to making it more static :)

**Comment by LouManglass** [comment]
I would approve this based on function, given our call and the subsequent changesets I pushed up. That's separate from Dev's comments, above.

**Pull request merged**